src/components/ShareBlock/ShareBlock.tsx

```diff
@@ -85,80 +85,82 @@ const ShareBlock = ({
-                    }}
-                  >
-                    <LinkedinIcon
-                      size={40}
-                      round={false}
-                      bgStyle={{ fill: 'auto' }}
-                    />
-                  </LinkedinShareButton>
-                </StyledShareButton>
-              </ShareButtonContainer>
-              <EmbedPrompt>
-                Or <span onClick={onClickEmbed}>embed on your website</span>
-              </EmbedPrompt>
-            </SocialTextArea>
-          </SocialTextAreaWrapper>
-          <SocialMockupWrapper>{SocialPreview}</SocialMockupWrapper>
-        </ShareRowContentArea>
-      </ShareRow>
+      {!region && (
+        <ShareRow newsletter={false}>
+          <ShareRowContentArea
+            $isMatchingProjectionsRoute={isMatchingProjectionsRoute !== null}
+          >
+            <SocialTextAreaWrapper id="shareblock">
+              <SocialTextArea>
+                <ShareInstructionHeader>
+                  Share COVID risk in your community
+                </ShareInstructionHeader>
+                <ShareInstructionBody>
+                  Share real-time, local COVID data with your neighbors,
+                  friends, and family.
+                </ShareInstructionBody>
+                <ShareButtonContainer reflow>
+                  <StyledShareButton variant="contained" color="#3b5998">
+                    <FacebookShareButton
+                      url={url}
+                      quote={quote}
+                      beforeOnClick={() => {
+                        trackShare('facebook');
+                        return Promise.resolve();
+                      }}
+                    >
+                      <FacebookIcon
+                        size={40}
+                        round={false}
+                        bgStyle={{ fill: 'auto' }}
+                      />
+                    </FacebookShareButton>
+                  </StyledShareButton>
+                  <StyledShareButton variant="contained" color="#00acee">
+                    <TwitterShareButton
+                      url={url}
+                      title={quote}
+                      hashtags={[hashtag]}
+                      beforeOnClick={() => {
+                        trackShare('twitter');
+                        return Promise.resolve();
+                      }}
+                    >
+                      <TwitterIcon
+                        size={40}
+                        round={false}
+                        bgStyle={{ fill: 'auto' }}
+                      />
+                    </TwitterShareButton>
+                  </StyledShareButton>
+                  <StyledShareButton variant="contained" color="#007fb1">
+                    <LinkedinShareButton
+                      url={url}
+                      title={quote}
+                      // @ts-ignore: seems to not be available for linkedin?
+                      hashtags={[hashtag]}
```

Comment:
We are not using LinkedIn anymore, are we?

Reply:
Ah you're right! For some reason I kept thinking the whole share block will be completely removed anyway but turns out we will still have it on the homepage.